### DISCUSSION TITLE
Welcome, devStack03!

### DISCUSSION THREAD

**Discussion by techelevate09** [ParanoidUser/yolo#1222] [Subscribe]
subscription

**Comment by ParanoidUser** [comment]
Ahoy @devStack03, thank you for your interest in the project! To get a Yolo badge, simply follow the steps in https://github.com/ParanoidUser/yolo/discussions/18.

Note: Don't forget to assign a reviewer on step 4. 🔭

Another note: You should receive a repo collaborator invite that allows you to merge PRs (steps 5-6).

Let me know if you have any questions. Welcome aboard! 🚢⭐
